pybossa/jobs.py

```diff
@@ -958,11 +958,19 @@ def export_tasks(current_user_email_addr, short_name,
             with export_fn(project, ty, expanded, filters, disclose_gold) as fp:
                 filename = fp.filename
                 content = fp.read()
-            data = project.info
-            bucket_name = current_app.config.get('EXPORT_BUCKET')
-            max_size = current_app.config.get('EXPORT_MAX_SIZE', float('Inf'))
 
-            if len(content) > max_size and bucket_name:
+            bucket_name = current_app.config.get('EXPORT_BUCKET')
+            max_email_size = current_app.config.get('EXPORT_MAX_EMAIL_SIZE', float('Inf'))
+            max_s3_upload_size = current_app.config.get('EXPORT_MAX_UPLOAD_SIZE', float('Inf'))
```

Comment:
Re `EXPORT_MAX_EMAIL_SIZE` and `EXPORT_MAX_UPLOAD_SIZE` 
1. can they be consolidated to one
2. can we also update `settings_local.py.tmpl` describing with comments what there values are to be determined from

Reply:
They must be 2 separate values, for example:

`EXPORT_MAX_EMAIL_SIZE` set to 5 mb for a max email attachment of 5 mb
`EXPORT_MAX_UPLOAD_SIZE` set to 100mb for a max s3 upload size of 100 mb.